### PR TITLE
Create header of the mobile version

### PR DESCRIPTION
**Pull request by thasipablo**
Create main files:

- .html file
- .css file

Define color variables:

- primary color
- secondary color
- highlight color

Define general CSS styles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          <li class="socials-item"><a href="https://github.com/thasipablo"><img src="images/github.svg" alt="github icon"></a></li>
+          <li class="socials-item"><a href="#"><img src="images/angellist.svg" alt="angellist icon"></a></li>
+        </ul>
+        <div class="banner-scroll-btn">
+          <button>
+            <img src="images/chevron_down.svg" alt="chevron down icon">
+          </button>
+        </div>
+      </div>
+    </section>
```
Comment (Kamba56) [inline index.html:48]: - [ ] As demanded by project requirements, kindly align contents of the headline section using a flexbox.
image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @thasipablo, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great UI implementation👏👏👏
- [x] 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (thasipablo) [reply index.html:48]: Thank you for your review.
It's now fixed

Comment (Ofarouq310) [inline index.html:48]: [OPTIONAL]
- [ ] Kindly wrap your whole `section` with the `main` tag.

---

- Kindly take a look at the picture which displays the tags that have to be used inside the `body`:
  -  First of all the `header`, which in some cases includes the `nav` inside of it, like our portfolio, and in some cases it doesn't.
  - Secondly, which is not displayed in the picture, the `main` tag, which wraps everything related to your visible content (sections, articles, divs, images, etc...).
  - Last but not least, the `footer`.
  
- All in all, your `body` should consist of 3 tags: `header`, `main`, and `footer`.

    image

File: README.md [hunk 2]
```diff
@@ -133,4 +134,10 @@ If you like this project fork it and make a PR, open an issue or invite me drink
 
 I would like to thank all my code partners, and all reviewers
 
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
+
```
Comment (Ofarouq310) [inline README.md:142]: [OPTIONAL]
- [ ] The link in the `LICENSE` section has to redirect the user to the `LICENSE` listed in your project files, please fix it, and customize the `LICENSE` in your project files with the right `username` and `year`.
- This [tutorial](https://www.youtube.com/watch?v=8Dm7XXvtqXI&t=1s&pp=ygULYWRkIGxpY2Vuc2U%3D) helps you to add a `LICENSE` in your repository, remember to choose the `MIT` license.

**Review by Ofarouq310 — APPROVED** [review]
Hi @thasipablo, ✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇  

giphy

## Highlights
✅ Descriptive PR title.
✅ Descriptive commit messages.
✅ No linters errors.
✅ Followed GitHub flow.
✅ Good looking README file.

## Optional suggestions

- [ ] Kindly make sure to have your commit messages capitalized.



- [ ] As for the PR description, you have to summarize what you implemented in the project using a bullet list, a checklist, or a numbers list, in your case it could be something like:
    - Added a header section to the portfolio,
    - Added a headline section with an introduction paragraph.
    - etc...
     
or anything that has the same intent of explaining the new additions to your project.


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (thasipablo) [reply README.md:142]: Thank you

File: README.md [hunk 3]
```diff
@@ -41,12 +41,13 @@ After you're finished please remove all the comments and instructions!
   - [🤝 Contributing ](#-contributing-)
   - [⭐️ Show your support ](#️-show-your-support-)
   - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
```
Comment (thasipablo) [inline README.md:44]: Good work

**Review by thasipablo — COMMENTED** [review]
Keep it up

**Pull request merged**